docs/components/board/_index.md

```diff
@@ -0,0 +1,1534 @@
+myBoard.SetPowerMode(context.Background(), boardpb.PowerMode_POWER_MODE_OFFLINE_DEEP, nil)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## `GPIOPin` API
+
+### Set
+
+Set the digital signal output of this pin to low (0V) or high (active, >0V).
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `high` [(bool)](https://docs.python.org/3/library/stdtypes.html#bltin-boolean-values): If `true`, set the state of the pin to high.
+If `false`, set the state of the pin to low.
+- `extra` [(Optional[Dict[str, Any]])](https://docs.python.org/library/typing.html#typing.Optional): Extra options to pass to the underlying RPC call.
+- `timeout` [(Optional[float])](https://docs.python.org/library/typing.html#typing.Optional): An option to set how long to wait (in seconds) before calling a time-out and closing the underlying RPC call.
+
+**Returns:**
+
+- None
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/board/index.html#viam.components.board.Board.GPIOPin.set).
+
+```python
+my_board = Board.from_robot(robot=robot, name="my_board")
+
+# Get the GPIOPin with Pin Number 15 (GPIO 22 on Raspberry Pi 4).
+pin = await my_board.GPIO_pin_by_name(name="15")
+
+# Set the pin to high.
+await pin.set(high="true")
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `Context` [(Context)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+- `high` [(bool)](https://pkg.go.dev/builtin#bool): If `true`, set the state of the pin to high.
+If `false`, set the state of the pin to low.
+- `extra` [(map[string]interface{})](https://pkg.go.dev/google.golang.org/protobuf/types/known/structpb): Extra options to pass to the underlying RPC call.
+
+**Returns:**
+
+- `error` [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/components/board#GPIOPin).
+
+```go
+myBoard, err := board.FromRobot(robot, "my_board")
+
+// Get the GPIOPin with Pin Number 15 (GPIO 22 on Raspberry Pi 4).
+pin, err := myBoard.GPIOPinByName("15")
+
+// Set the pin to high.
+err := pin.Set(context.Background(), "true", nil)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Get
+
+Get if the digital signal output of this pin is high (active, >0V).
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `extra` [(Optional[Dict[str, Any]])](https://docs.python.org/library/typing.html#typing.Optional): Extra options to pass to the underlying RPC call.
+- `timeout` [(Optional[float])](https://docs.python.org/library/typing.html#typing.Optional): An option to set how long to wait (in seconds) before calling a time-out and closing the underlying RPC call.
+
+**Returns:**
+
+- `high` [(bool)](https://docs.python.org/3/library/stdtypes.html#bltin-boolean-values): If `true`, the state of the pin is set to high.
+If `false`, the state of the pin is set to low.
```

Comment:
```suggestion
- `high` [(bool)](https://docs.python.org/3/library/stdtypes.html#bltin-boolean-values): If `true`, the state of the pin is set to high.
If `false`, the state of the pin is set to low.
```
```suggestion
- `high` [(bool)](https://docs.python.org/3/library/stdtypes.html#bltin-boolean-values): If `true`, the state of the pin is high.
If `false`, the state of the pin is low.
```
I fear they'd think "set to" means this method will change the state

Reply:
Ahh good point!